docs/controls/InAppNotification.md

```diff
@@ -84,6 +84,15 @@ ExampleInAppNotification.Dismiss();
 If you want to fully customize the in-app notification, you will see that the Dismiss button is still visible.
 To hide it, simply set the property to `ShowDismissButton="False"`.
 
+### StackMode
+
+By default, each time you display an in-app notification using the same control, each notification will replace the previous one.
+You can change this behavior with one of these values:
+
+* `Replace` - default mode, replace previous notification
+* `QueueBehind` - store every notifications to show, when you dismiss a notification the remaining ones will be displayed successively
```

Comment:
will be displayed in succession.

Also should this be formatted as a table?

Reply:
I can. But this documentation page does not follow the guidelines for the moment.

Reply:
Let it be like this. I will change it in #1748. Currently, I have updated the old doc. You can see it [here](https://github.com/Vijay-Nirmal/UWPCommunityToolkit/blob/e30871817ddea21d57c1b21503db8df82fd87d0a/docs/controls/InAppNotification.md)